src/System.IO.Abstractions.TestingHelpers/System.IO.Abstractions.TestingHelpers.csproj

```diff
@@ -4,6 +4,7 @@
         <RootNamespace>System.IO.Abstractions.TestingHelpers</RootNamespace>
         <Description>A set of pre-built mocks to help when testing file system interactions.</Description>
         <TargetFrameworks>net5.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+        <PackageIconUrl>https://avatars1.githubusercontent.com/u/40829877</PackageIconUrl>
```

Comment:
This property seems to be obsolete [according to the build log](https://github.com/System-IO-Abstractions/System.IO.Abstractions/pull/742/checks?check_run_id=3141881403#step:4:23). Could you change this to follow current guidelines?